lib/handlers/quote/quote.ts

```diff
@@ -453,8 +455,8 @@ export class QuoteHandler extends APIGLambdaHandler<
       gasUseEstimateQuoteDecimals: estimatedGasUsedQuoteToken.toExact(),
       gasUseEstimate: estimatedGasUsed.toString(),
       gasUseEstimateUSD: estimatedGasUsedUSD.toExact(),
-      simulationStatus: simulationStatusToString(simulationStatus, log),
-      simulationError: (simulationStatus == SimulationStatus.Succeeded || simulationStatus == SimulationStatus.Unattempted) ? false : true,
+      simulationStatus: simulationStatus ? simulationStatusToString(simulationStatus, log) : undefined,
+      simulationError: simulationStatus === undefined || simulationStatus == SimulationStatus.Succeeded ? false : true,
```

Comment:
Just trying to get some more context here because I'm not super familiar with this code. Why would `simulationError` be `false` if `simulationStatus` is `undefined`? Is that because we are skipping the simulation so we are okay with the undefined status?

Reply:
yeah that's right, simulationStatus will be undefined if the client didn't request a simulation, so it is fine.